src/AuthContext.tsx

```diff
@@ -185,6 +187,7 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
           })
           .finally(() => {
             window.history.replaceState(null, '', window.location.pathname) // Clear ugly url params
+            setLoginInProgress(false)
```

Comment:
seems like an unrelated (yet important) fix?

Reply:
Ah yes, we considered adding another (related) fix, and while doing that we moved this here (as an unrelated fix), and then we removed the related fix, leading to only the unrelated one sticking. I'll make a seperate PR for this alone.

Reply:
https://github.com/soofstad/react-oauth2-pkce/pull/67